netket/sampler/numpy/metropolis_hastings.py

```diff
@@ -127,4 +126,4 @@ def __next__(self):
     @property
     def acceptance(self):
         """The measured acceptance probability."""
-        return _mean(self._accepted_samples) / _mean(self._total_samples)
```

Comment:
I think the correct fix would be

```
    def acceptance(self):		     def acceptance(self):
         """The measured acceptance probability."""		         """The measured acceptance probability."""
         return _sum_inplace(self._accepted_samples) /self._total_samples		
```

Reply:
total_samples is already summed over MPI nodes?

Reply:
Together with register the dispatch rule that I posted above

Reply:
@gcarleo nope, you are right. I fixed it

Reply:
Ok, I cannot modify or approve this suggestion. Also, _sum_inplace has to be imported from stats, right?